### ISSUE TITLE
dgram error on `t2 list`

### ISSUE THREAD

**Issue by Frijol** [tessel/t2-cli#343]
When I run CLI on master:

```sh
➜  t2-cli git:(master) t2 list
INFO Searching for nearby Tessels...
dgram.js:455
    throw new Error('Not running'); // error message from dgram_legacy.js
          ^
Error: Not running
    at Socket._healthCheck (dgram.js:455:11)
    at Socket.address (dgram.js:365:8)
    at SendWrap.callback (/Users/frijol/t2-cli/node_modules/mdns-js/lib/networking.js:167:50)
    at SendWrap.afterSend [as oncomplete] (dgram.js:341:8)
```

**Comment by johnnyman727** [comment]
Yeah - I've seen this as well (and I think @rwaldron experienced it too). I think it's thrown when Bonjour isn't running on OSX (it's responsible for mDNS).

**Comment by mcollina** [comment]
Just so that you know, which version of node are you using? Node < 4 has
some serious issues in the dgram module regarding error handling.


Il giorno dom 27 set 2015 alle 19:57 Jon <notifications@github.com> ha
scritto:

**Comment by johnnyman727** [comment]
@mcollina currently at 0.12.X. The `usb` module isn't compatible with 4.0 yet - it's stuck on [this issue](https://github.com/nonolith/node-usb/pull/90). The issue that @Frijol is reporting here is difficult to reproduce but we could potentially dive deeper b/c you don't need `usb` to start debugging mDNS bugs.

So, next time anyone sees this, comment out [this line](https://github.com/tessel/t2-cli/blob/master/lib/discover.js#L1), and [these lines](https://github.com/tessel/t2-cli/blob/master/lib/discover.js#L27-L29). Then update to 4.0 and see if the issue still occurs. If it does not, this will be resolved when we are compatible with 4.0.

**Comment by johnnyman727** [comment]
Closing this for now because it is very hard to reproduce - I have not seen it for months.
